superset/sqllab/api.py

```diff
@@ -340,13 +340,15 @@ def get_results(self, **kwargs: Any) -> FlaskResponse:
         key = params.get("key")
         rows = params.get("rows")
         result = SqlExecutionResultsCommand(key=key, rows=rows).run()
+
+        payload = json.dumps(
+            result,
+            default=utils.pessimistic_json_iso_dttm_ser,
+            ignore_nan=True,
+        )
         # return the result without special encoding
         return json_success(
-            json.dumps(
```

Comment:
Why not keep this inline given `payload` is a single use variable.

Reply:
on this I'm happy either way, but thought the change made things a bit more readable, maybe making it a tiny bit easier to read the stacktrace or debug. But I don't have strong opinions on single-use variable